Decred Wallet/Extensions/UIStackView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIStackView.swift
+//  Decred Wallet
+//
+//  Created by Sprinthub on 05/09/2019.
+//  Copyright © 2019 Decred. All rights reserved.
+//
+
+import UIKit
+
+extension UIStackView {
+    
+    func cornerRadius(_ radius: CGFloat){
```

Comment:
There is a lot going on here apart from just setting the corner radius. You might want to extract a few things out of this or rename the method to something more descriptive of all it does.

Reply:
I also noticed in the designs the decred logo is fixed in the navbar and not part of the overview page. When scrolling it shouldn't scroll together with the view.

Reply:
added comments to address observation 1 in e50fe14

Reply:
The fixed navbar with a fixed logo and dynamic text (switches from Overview to wallet amount and back to overview on scroll) is achievable when the nav is abstracted. this is being addressed here:  [https://github.com/quacklabs/dcrios/tree/navmenu-new-ui](url) as delegate methods to trigger the change will be added here as other views also modify the title label.